libexec/scoop-virustotal.ps1

```diff
@@ -155,15 +198,20 @@ if(is_scoop_outdated) {
 $apps_param = $apps
 
 if($apps_param -eq '*') {
-    $apps = applist (installed_apps $false) $false
-} else {
-    $apps = ensure_all_installed $apps_param
+    $apps = (applist (installed_apps $false) $false) | ForEach-Object {
+                ($a, $_global_flag_to_drop) = $_
+                $a
```

Comment:
What does this `$a` do?

Reply:
`applist` returns a list of tuples: the application name and a flag saying whether the application is installed globally.  For my purposes, I only need the application name.

It is a way to transform the list `@(@("app1", flag1), @("app2", flag2) )` into `@("app1", "app2")`.

The change was needed because I now have two sources of the list of apps to check with the VirusTotal service `installed_apps` (returning a list of tuples, as discussed above) and `install_order` (see [line 215 (was 208)](https://github.com/pcrama/scoop/blob/b3221133785be10b394fb6cd7ce69885a252e6e4/libexec/scoop-virustotal.ps1#L215)) to also include the list of dependencies in the scan.

The flag returned by `applist` for each application used to be dropped from that tuple in the final loop (see change from [(previously) line 166](https://github.com/pcrama/scoop/blob/af41fc6efb0fdf65943b8639ca5d44a406445452/libexec/scoop-virustotal.ps1#L166) to [line 221](https://github.com/pcrama/scoop/blob/b3221133785be10b394fb6cd7ce69885a252e6e4/libexec/scoop-virustotal.ps1#L221))

As you can see I pushed a comment in the source code about this.

If there's a better way in PowerShell to do this transformation, I'm all ears.

Thank you for looking at my PR!

Reply:
`return $a` would have made it more clear I guess. I simply can't wrap my head around the fact that PowerShell returns the last variable by itself ... 😁